### DISCUSSION TITLE
Copilot won't Run in VS Code - how do I fix it?

### DISCUSSION THREAD

**Discussion by jasonburrows** [community/community#50840] [Copilot Conversations]
### Select Topic Area

Question

### Body

I have the most recent version of VS Code and Github Copilot.  I install the extension.  It does nothing.  A copilot icon appears in the lower right hand corner quickly, and then is replaced by a spinning wheel that never goes away.

If I open the output for copilot, it is totally blank.  No errors are provided.  No dialogs asking me to log in or anything ever come up.  Eventually, after a long period of time, the "Runtime Status" appears in the extension configuration page and it says "Not yet activated".

I have restarted VS Code, uninstall and re-installed copilot, rebooted, done the magic danse.

Note that I do have it working properly on another dev box, and there it just happened with no issues.  Both same version of macOS and everything else.

I've seen similar threads to this and nothing there applies to me (no proxy for example).

What do I do?

**Comment by jasonburrows** [comment]
...so in the end I had to open a new project with nothing in it, and then reinstall the plugin.  Then I clicked on the spinning wheel and this time it brought up the dialog to login to github.  I'm not sure what it was as I tried with different projects open, but getting past it I needed to have nothing open at all. 

Anyway - works now.

**Reply by BitCalSaul** [reply]
same here. It works when I switch to another workspace even it's still the same server.
